parquet-hadoop/src/main/java/org/apache/parquet/hadoop/example/ExampleParquetWriter.java

```diff
@@ -113,11 +110,6 @@ public Builder withType(MessageType type) {
       return this;
     }
 
-    public Builder withExtraMetaData(Map<String, String> extraMetaData) {
```

Comment:
This method is now available through the `ParquetWriter.Builder` superclass.

Reply:
Is it possible to add an overload here to suppress the complaint of japicmp? That's more preferable than an exclusion.

Reply:
great idea! updated + removed the exclusion.